sale_order_import_ubl/wizard/sale_order_import.py

```diff
@@ -26,7 +26,9 @@ def parse_xml_order(self, xml_root, detect_doc_type=False):
             else:
                 return self.parse_ubl_sale_order(xml_root)
         else:
-            return super(SaleOrderImport, self).parse_xml_order(xml_root)
+            return super(SaleOrderImport, self).parse_xml_order(
+                xml_root, detect_doc_type
```

Comment:
Well, this is not broken but it will have a strange behavior if you have installed several modules handling SO import based on the order their override is executed.

Also, in v14 I don't find any other module but this one, checking for `detect_doc_type`. I guess you overridden it as well?

Which error in particular have you faced?